rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -592,6 +591,11 @@ namespace rsx
 			m_guide_interval = guide_interval;
 		}
 
+		u16 graph::get_height() const
+		{
+			return h + 4 + m_label.h;
+		}
```

Comment:
Just wondering where the 4 comes from.
Could it be possible to retrieve that value from a constant or other variable?

Reply:
yea